tools/pt/libipt/ptxed/CMakeLists.txt

```diff
@@ -38,6 +39,7 @@ include_directories(SYSTEM
 
 link_directories(
   ${XED_LIBDIR}
+  ../../obj/wkit/lib
```

Comment:
This line is no longer needed once you fix the relative paths in `tools/pt/Makefile`

```suggestion
```